xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-ui/src/main/resources/RTFrontend/Ajax.xml

```diff
@@ -74,11 +74,23 @@
   #if ("$!{request.convertHTML}" == '1')
     #set ($xdom = $services.rendering.parse($currentContent, 'xhtml/1.0'))
     #set ($currentContent = $services.rendering.render($xdom, $nextDocument.syntax.toIdString()))
+    ## FIXME: This requires programming rights.
```

Comment:
Maybe create issues for the FIXME (and mention their ids in the comments) if we plan to fix them.

Reply:
I removed the Ajax service. Getting rid of these rights issues.
Thanks.